app/design/frontend/base/default/template/catalog/category/title.phtml

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/**
+ * Maho
+ *
+ * @package     base_default
+ * @copyright   Copyright (c) 2006-2020 Magento, Inc. (https://magento.com)
+ * @copyright   Copyright (c) 2021-2023 The OpenMage Contributors (https://openmage.org)
+ * @copyright   Copyright (c) 2024-2025 Maho (https://mahocommerce.com)
+ * @license     https://opensource.org/licenses/afl-3.0.php  Academic Free License (AFL 3.0)
+ */
+
+/** @var Mage_Catalog_Block_Category_Title $this */
+?>
+<?php if ($title = $this->getTitle()): ?>
+<div class="page-title category-title">
+    <?php if ($this->getShowRssLink()): ?>
+        <a href="<?= $this->escapeHtml($this->buildRssLink()) ?>" class="link-rss"><?= $this->escapeHtml($this->getRssText()) ?></a>
```

Comment:
`Mage_Core_Model_Url::getUrl()` already escapes, so not needed. :)

```suggestion
        <a href="<?= $this->buildRssLink() ?>" class="link-rss"><?= $this->escapeHtml($this->getRssText()) ?></a>
```

Reply:
wow, what an eye!